app/components/Modal/BorrowModal.jsx

```diff
@@ -438,19 +452,22 @@ class BorrowModalContent extends React.Component {
                             <div>
                                 <div className={collateralRatioClass}>
                                     <Translate component="label" content="borrow.coll_ratio" />
-                                    <input min="0" max="6" step="0.01" onChange={this._onRatioChange.bind(this)} value={collateral_ratio} type="range" disabled={!short_amount}/>
+                                    <input min="0" max="6" step="0.01" onChange={this._onRatioChange.bind(this)} value={collateral_ratio} type="range" />
                                     <div className="inline-block">{utils.format_number(collateral_ratio, 2)}</div>
                                     {errors.below_maintenance || errors.close_maintenance ? <div style={{maxWidth: "calc(100% - 50px)"}} className="float-right">{errors.below_maintenance}{errors.close_maintenance}</div> : null}
                                 </div>
                             </div>
                           ) : null}
-                        <div className="no-padding grid-content button-group no-overflow">
+                        <div className="no-padding grid-content button-group no-overflow float-left">
                             <div onClick={this._onSubmit.bind(this)} href className={buttonClass}><Translate content="borrow.adjust" /></div>
                             <div onClick={(e) => {e.preventDefault(); this.setState(this._initialState(this.props));}} href className="button info"><Translate content="wallet.reset" /></div>
                             {/*<Trigger close={this.props.modalId}>
                                 <div className="button"><Translate content="account.perm.cancel" /></div>
                             </Trigger>*/}
-                            <div href className="float-right button info" onClick={this._maximizeCollateral.bind(this)}>Maximize Collateral</div>
+                        </div>
+                        <div className="no-padding grid-content button-group no-overflow float-right">
+                            <div href className="button info" onClick={this._maximizeDebt.bind(this)}>Maximize Debt</div>
```

Comment:
Please add a translation key for this instead of hardcoding the English value. Use the "borrow" group like the other translations in this component. If you can add one for Maximize Collateral as well that would be great.